### ISSUE TITLE
Tarantool modules can't be installed on MacOS

### ISSUE THREAD

**Issue by lenkis** [tarantool/tarantool#2015]
Can't install modules from either deb/rpm packages or luarocks at MacOS (Sierra 10.12.2).

**Comment by rtsisyk** [comment]
macOS doesn't support DEB/RPM packages. Please try to use LuaRocks instead.

**Comment by bigbes** [comment]
macOS have a problem with LuaRocks too. It must be build and shipped with
Tarantool (in homebrew). Building LuaRocks for Tarantool depends on ticket
with Lua CLI flags: https://github.com/tarantool/tarantool/issues/1265

2017-01-10 13:52 GMT+03:00 Roman Tsisyk <notifications@github.com>:

> macOS doesn't support DEB/RPM packages. Please try to use LuaRocks instead.
>



With best regards, Eugene.

**Comment by kostja** [comment]
Duplicate of gh-2067
